winrt-related-src/schemas/appxpackage/uapmanifestschema/element-desktop7-shortcut.md

```diff
@@ -81,13 +81,13 @@ The *File* path format is a path relative to one of the following constants that
 | Constant | Directory |
 |----------|-----------|
 | Common Programs | C:\ProgramData\Microsoft\Windows\Start Menu\Programs |
-| Programs | C:\Users\<username>\AppData\Roaming\Microsoft\Windows\Start Menu\Programs |
-| Desktop | C:\Users\<username>\Desktop |
+| Programs | C:\Users\\<username>\AppData\Roaming\Microsoft\Windows\Start Menu\Programs |
+| Desktop | C:\Users\\<username>\Desktop |
 
-So, for example the following value for *File* creates a shortcut with the path `C:\Users\<username>\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Subfolder\Shortcut.exe.lnk`.
+So, for example the following value for *File* creates a shortcut with the path `C:\Users\<username>\Desktop\Shortcut.lnk`.
 
 ```xml
-File="$$(Programs)\Subfolder\Shortcut.exe.lnk"
+File="[{Desktop}]\Shortcut.lnk"
```

Comment:
```
File="$(Desktop)\Shortcut.lnk"
```

Use the `$(x)` syntax. The `[{x}]` syntax is a legacy for compat with early work, and thus may be deprecated and removed someday.

Reply:
If _early work_ means something that was GA at some point, then we probably should stick with the lowest common denominator here and/or add a note to this page offering both options?